webapp/security/views.py

```diff
@@ -499,202 +498,102 @@ def cve_index():
     """
     # Query parameters
     query = flask.request.args.get("q", "").strip()
-    priority = flask.request.args.get("priority")
-    package = flask.request.args.get("package")
+    priority = flask.request.args.get("priority", default="", type=str)
+    package = flask.request.args.get("package", default="", type=str)
     limit = flask.request.args.get("limit", default=20, type=int)
     offset = flask.request.args.get("offset", default=0, type=int)
     component = flask.request.args.get("component")
+    version = flask.request.args.get("version", default="", type=str)
+    status = flask.request.args.get("status", default="", type=str)
     versions = flask.request.args.getlist("version")
     statuses = flask.request.args.getlist("status")
 
-    is_cve_id = re.match(r"^CVE-\d{4}-\d{4,7}$", query.upper())
-
-    if is_cve_id and db_session.query(CVE).get(query.upper()):
-        return flask.redirect(f"/security/{query.lower()}")
-
-    all_releases = (
-        db_session.query(Release)
-        .order_by(desc(Release.release_date))
-        .filter(Release.codename != "upstream")
-        .all()
-    )
-
-    releases_query = db_session.query(Release).order_by(Release.release_date)
-
-    if versions and not any(a in ["", "current"] for a in versions):
-        releases_query = releases_query.filter(Release.codename.in_(versions))
-    else:
-        releases_query = releases_query.filter(
-            or_(
-                Release.support_expires > datetime.now(),
-                Release.esm_expires > datetime.now(),
-            )
-        ).filter(Release.codename != "upstream")
-
-    releases = releases_query.all()
-
-    should_filter_by_version_and_status = (
-        versions and statuses and len(versions) == len(statuses)
+    # get cves and total results
+    cves_response = security_api.get_cves(
+        query=query,
+        priority=priority,
+        package=package,
+        limit=limit,
+        offset=offset,
+        component=component,
+        version=version,
```

Comment:
@mtruj013 Didn't debug the API response very well, but I'm gonna take a guess:

If you select two Ubuntu versions the url is like  `...&version=jammy&status=&version=focal&status=...`

Since `version` is only the first one (`flask.request.args.get("version", default="", type=str)` only returns one) could it be that you're requesting only one version from the API and that's why the table is empty for the others?